docs/framework/wcf/generating-data-type-classes-from-xml.md

```diff
@@ -16,11 +16,11 @@ ms.author: "erikre"
 manager: "erikre"
 ---
 # Generating Data Type Classes from XML
-[!INCLUDE[net_v45](../../../includes/net-v45-md.md)] includes a new feature to generate data type classes from XML. This topic describes how to automatically generate data types for the MSDN Library RSS feed.  
+[!INCLUDE[net_v45](../../../includes/net-v45-md.md)] includes a new feature to generate data type classes from XML. This topic describes how to automatically generate data types for the .NET Blog RSS feed.  
   
-### Obtaining the XML from the MSDN Library RSS feed  
+### Obtaining the XML from the .NET Blog RSS feed  
   
-1.  In Internet Explorer, navigate to the [MSDN RSS feed](http://go.microsoft.com/fwlink/?LinkId=225209).  
+1.  In Internet Explorer, navigate to the [.NET Blog RSS feed](https://blogs.msdn.microsoft.com/dotnet/feed/).  
```

Comment:
since we changed the RSS feed here, we need to add the other class names (or mention we have more) in line 37 as I mentioned in the issue